qt/python/mantidqt/widgets/fitpropertybrowser/addfunctiondialog/presenter.py

```diff
@@ -0,0 +1,32 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, unicode_literals)
+
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtWidgets import QDialog
+
+from .view import AddFunctionDialogView
+
+
+class AddFunctionDialog(QDialog):
```

Comment:
Usually the presenter doesn't inherit from a Qt widget - is this particularly required here for some reason?

Reply:
I initially did it so the signal would work correctly but I've moved that to the view so it's not needed now